docs/reference/commandline/run.md

```diff
@@ -2,6 +2,8 @@
 title: "run"
 description: "The run command description and usage"
 keywords: "run, command, container"
+redirect_from: 
+- /v1.11/engine/reference/commandline/run/
```

Comment:
Same for this one

https://docs.docker.com/v1.11/engine/reference/commandline/run/

Reply:
Same answer as above.